test/src/test/java/mil/nga/giat/geowave/test/service/GeoWaveIngestGeoserverIT.java

```diff
@@ -47,7 +47,7 @@
 
 	private static final Logger LOGGER = Logger.getLogger(GeoWaveIngestGeoserverIT.class);
 
-	private static final String WORKSPACE = "testomatic";
+	private static final String WORKSPACE = "testomatic2";
```

Comment:
Looking back at it, I probably got that error because I needed to delete the testomatic folder under target before running that test individually.